[GSoD 2021] Added CONTRIBUTING.md

iamrajiv opened the pull request:
Related: wechaty/wechaty.js.org#1012

![Screenshot 2021-07-06 at 9 33 34 PM](https://user-images.githubusercontent.com/42106787/124632339-df286700-dea1-11eb-88d9-6119c6939306.jpg)
![Screenshot 2021-07-06 at 9 33 48 PM](https://user-images.githubusercontent.com/42106787/124632319-dc2d7680-dea1-11eb-946a-32f20f6d555a.jpg)

Rohitesh-Kumar-Jain reviewed (commented):
pass CI test so that we can continue reviewing

iamrajiv commented:
@Rohitesh-Kumar-Jain see this https://github.com/wechaty/wechaty.js.org/issues/1012#issuecomment-882695556

huan reviewed (changes requested):
Thanks for creating this CONTRIBUTING file!

As we have already put a contributing docs at our website https://wechaty.js.org/docs/contributing/ , so I'd like to suggest that we can move all the content to our website, and put the website link of https://wechaty.js.org/docs/contributing/ in this file.

Please feel free to improve the https://wechaty.js.org/docs/contributing/ as you need, and then change this file content to just link to our website.

Thank you very much.

iamrajiv commented:
@huan I have updated to contributing file of this PR and added the https://wechaty.js.org/docs/contributing/ link. I have made PR for improvement of contributing  guidelines here #wechaty/wechaty.wechaty.js.org#1276, please review

huan commented:
Thank you very much!
